Source/WebCore/page/LocalDOMWindow.cpp

```diff
@@ -1149,6 +1149,7 @@ void LocalDOMWindow::stop()
     if (!frame)
         return;
 
+    SetForScope isStopped { m_isStopping, true };
```

Comment:
isStopping?
Also, shouldn't we need to write it as `m_isStopping, true, false`?